README.md

```diff
@@ -53,6 +54,22 @@ By supplying `--all` to c8, all files in directories specified with `--src` (def
 and `--exclude` flag checks, will be loaded into the report. If any of those files remain uncovered they will be factored
 into the report with a default of 0% coverage.
 
+## Pre-instrumented versus just-in-time instrumented codebases
+
+### Source map files versus inline source maps
+
+Just-in-time instrumented codebases will often insert source maps inline with the `.js` code they generate at runtime (e.g, `@babel/register` can be configured to insert a source map footer).
+
+Pre-instrumented codebases, e.g., running `tsc` to generate `.js` in a build folder, may generate either inline source maps, or a separate `.map` file stored on disk.
+
+`c8` can handle loading both types of source maps.
+
+### Exclude after remap
+
+For independent projects, feel free to set `--exclude-after-remap` as `true` or `false`, the only difference is that the exclusions are applied to _**source**_ scripts or _**executable**_ scripts.
+
+For huge 'shared codebase' such as monorepo, your test suites can use common libs as workspace node_modules and _**executable**_ scripts is a single bundle, set `--exclude-after-remap` to true may help to set exclusions more precisely.
```

Comment:
```suggestion
Depending on the size and configuration of your project, it may be preferable to apply exclusion logic either before or after source-maps are used to remap compiled to original source files.

`--exclude-after-remap` is used to control this behaviour.
```

Reply:
LGTM